src/main/java/com/example/bookstore/config/Config.java

```diff
@@ -0,0 +1,13 @@
+package com.example.bookstore.config;
+
+import java.time.format.DateTimeFormatter;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
why did you used this annotation for regular constants class? there's no sense in it
```suggestion
```